ouroboros-consensus/ouroboros-consensus.cabal

```
@@ -685,8 +685,8 @@ test-suite storage-test
     , ouroboros-network-mock
     , pretty-show
     , QuickCheck
-    , quickcheck-dynamic
-    , quickcheck-lockstep
+    , quickcheck-dynamic        >=3.3.1
+    , quickcheck-lockstep       >=0.3
```

I was going to suggest adding upper bounds:

```suggestion
    , quickcheck-dynamic        ^>=3.3.1
    , quickcheck-lockstep       ^>=0.3
```

But in tests I think it is probably unnecessary.